google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/SelectConfigDestinationFolderDialog.java

```diff
@@ -53,6 +55,25 @@ public SelectConfigDestinationFolderDialog(@Nullable Project project) {
         project,
         FileChooserDescriptorFactory.createSingleFolderDescriptor()
     );
+
+    if (project != null && project.getBasePath() != null) {
+      String srcPath = project.getBasePath() + "/src";
+      String srcMainPath = project.getBasePath() + "/src/main";
+      String AppenginePath = project.getBasePath() + "/src/main/appengine";
+      String dockerPath = project.getBasePath() + "/src/main/docker";
+
+      if (fileType == ConfigFileType.APP_YAML && new File(AppenginePath).isDirectory()) {
```

Comment:
Just wanted to +1 Etan's suggestion. That's how I saw it working. We want
the user to be guided to placing the files in the standard location.

On Fri, Apr 8, 2016 at 12:34 PM, Etan Shaul <notifications@github.com>
wrote:

> In
> google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/SelectConfigDestinationFolderDialog.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/614#discussion_r59052977>
> :
>
> > @@ -53,6 +55,25 @@ public SelectConfigDestinationFolderDialog(@Nullable Project project) {
> >          project,
> >          FileChooserDescriptorFactory.createSingleFolderDescriptor()
> >      );
> > +
> > +    if (project != null && project.getBasePath() != null) {
> > +      String srcPath = project.getBasePath() + "/src";
> > +      String srcMainPath = project.getBasePath() + "/src/main";
> > +      String AppenginePath = project.getBasePath() + "/src/main/appengine";
> > +      String dockerPath = project.getBasePath() + "/src/main/docker";
> > +
> > +      if (fileType == ConfigFileType.APP_YAML && new File(AppenginePath).isDirectory()) {
>
> I wonder if it makes sense to default it to the canonical directories even
> if they don't already exist? In this case it should prompt the user to
> create them (I think this feature was added previously).
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/614/files/adbb4c8890c80496bbc12b06984bf849b80a2382#r59052977>
>